### PR TITLE
v0.1.0

### PR DESCRIPTION
**Pull request by bluquar**
> If your first release isn't embarrassingly bad, you're doing something wrong. - @lowellbander 

Currently deployed at [`http://app.dandelion.supplies`](http://app.dandelion.supplies) via manual deployment.

https://user-images.githubusercontent.com/4309735/146696941-58a48282-558e-476c-a8b2-36f6d71af69c.mov

### CODE REVIEW

File: .github/workflows/deploy-to-eb.yaml [hunk 1]
```diff
@@ -34,9 +34,12 @@ jobs:
         run: npm prune --production
 
       - name: Create secret environment variables
-        run:
-          echo "MONGODB_USER_0_PASSWORD=\"${{ secrets.MONGODB_USER_0_PASSWORD }}\"" > .env
-          echo "SEGMENT_ANALYTICS_WRITE_KEY=\"${{ secrets.SEGMENT_ANALYTICS_WRITE_KEY }}\"" >> .env
+        uses: SpicyPizza/create-envfile@v1
```
Comment (lowellbander) [inline .github/workflows/deploy-to-eb.yaml:37]: wat is `SpicyPizza` luhmao

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "community-aid-app",
-  "version": "0.0.0",
+  "version": "0.1.0",
```
Comment (lowellbander) [inline package.json:3]: YEEEEEEEET

**Review by lowellbander — APPROVED** [review]
VEE ZERO DOT ONE DOT ZERO

File: .github/workflows/deploy-to-eb.yaml (hunk 1 above)
Comment (bluquar) [reply .github/workflows/deploy-to-eb.yaml:37]: That's the person who made that action lol

**Pull request merged**